clusters/hive/pools/cvp/cvp-ocp-4-7-0-amd64-aws-us-east-1_clusterpool.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  name: ci-ocp-4-7-amd64-aws-us-east-1
+  namespace: cvp-cluster-pool
+  labels:
+    product: ocp
+    version: "4.7"
+    architecture: amd64
+    cloud: aws
+    owner: dpp
+    region: us-east-1
+spec:
+  baseDomain: hive.aws.ci.openshift.org
+  imageSetRef:
+    name: ocp-4.7.0-amd64
+  installConfigSecretTemplateRef:
+    name: install-config-aws-us-east-1
+  skipMachinePools: true
+  platform:
+    aws:
+      credentialsSecretRef:
+        name: cpaas-aws-credentials
```

Comment:
I'll need to check if this secret is available on the Hive cluster

Reply:
Please let me know if I can help somehow.

Reply:
How can we asure that?
I searched at _config
https://github.com/openshift/release/blob/master/core-services/ci-secret-bootstrap/_config.yaml#L749

Reply:
With pools, we go away with these bundled secrets. You need to make sure the appropriate cloud credentials (`.awscred`) is synced to this secret from Vault to the hive cluster. I don't know if you did that.

Reply:
I didn't check that, I presume I would need to be member of those creds or log in with account that has created them. Unfortunately I have none of those.